packages/core/src/shared/resourcefetcher/httpResourceFetcher.ts

```diff
@@ -22,15 +22,19 @@ export class HttpResourceFetcher implements ResourceFetcher<Response> {
      * @param {string} params.friendlyName If URL is not shown, replaces the URL with this text.
      * @param {Timeout} params.timeout Timeout token to abort/cancel the request. Similar to `AbortSignal`.
      * @param {number} params.retries The number of retries a get request should make if one fails
+     * @param {boolean} params.swallowError True if we want to swallow the request error, false if we want the error to keep bubbling up
      */
     public constructor(
         private readonly url: string,
         private readonly params: {
             showUrl: boolean
             friendlyName?: string
             timeout?: Timeout
+            swallowError?: boolean
```

Comment:
couple other naming ideas: throwOnError, supressError or propogateError

Reply:
is it different than `retryOnFail` ?